projects/js-packages/publicize-components/src/components/social-previews/mastodon.js

```diff
@@ -40,7 +38,6 @@ const MastodonPreview = props => {
 			description={ content }
 			customText={ message }
 			customImage={ customImage }
-			isSocialPost={ attachedMedia.length > 0 }
```

Comment:
Just for clarity, why are we removing this?

Reply:
It's not used anywhere.